New “Swift is…” and About page copy

daveverwer opened the pull request:
During the last meeting, we discussed having a `h1` with “Safe, Fast, Modern“ above the “Swift is…“ copy here, but after playing with it, I don’t think that feels quite right.

I’d be happy to change it if the consensus is to have that strong three-word lead-in, but it didn’t feel quite right to me.

This is what felt best to me:

![Screenshot 2023-07-03 at 19 41 52@2x](https://github.com/apple/swift-org-website/assets/5180/4a2474b8-afbf-437d-8a6a-615ff5f1de84)

Then I also tackled the "Safe, Fast, Expressive" section that begins the "About" page as I think they should line up. This copy *definitely* needs work, this is just a first draft. Let's discuss in this PR.

![Screenshot 2023-07-03 at 19 52 20@2x](https://github.com/apple/swift-org-website/assets/5180/1804c2d0-ddbe-4bd2-87a1-7e6fdf795c33)

tomerd commented:
lgtm, I would also consider the option of

Swift is a **general-purpose** programming language **approachable** for newcomers and **powerful** in expert hands. It is **quick** to write, **fast** to run, **modern**, and **safe**. 

in the "about" page, the "double bolding" seems redundant to me, but not a big deal.

alexandersandberg commented:
This looks great, Dave!

My only concern is the length of the `<h1>`. I think it's usually recommended to keep this under ~70 characters for SEO reasons. Now it's 164.

But as we keep the meta title as "Welcome to Swift.org", this might not be as much of an issue. I'm no SEO expert though.

As a side note, maybe we should update that meta title too while we're at it?

I think I agree with Tom as well that we don't need the bolding on `/about`.

daveverwer commented:
> Swift is a **general-purpose** programming language **approachable** for newcomers and **powerful** in expert hands. It is **quick** to write, **fast** to run, **modern**, and **safe**.

![Screenshot 2023-07-04 at 09 06 31@2x](https://github.com/apple/swift-org-website/assets/5180/ca9f21b5-3d51-4747-9b67-1a664901a891)

I agree, I like the ending of "quick, fast, modern, and safe".

For the about page, I replaced the opening paragraph with the new wording and removed the bolding.

![Screenshot 2023-07-04 at 09 15 45@2x](https://github.com/apple/swift-org-website/assets/5180/638683ee-2988-408b-be4f-c51939922922)

daveverwer commented:
> I think it's usually recommended to keep this under ~70 characters for SEO reasons. Now it's 164.

Yes, I considered this. The most concrete thing I found about how Google would treat this is that it may only pay attention to the first 50-80 characters of what we put in an `h1`.

I'm happy to change it to be a `p`, but in my opinion we're better with the first X characters of this long `h1` than no `h1` (or the other `h1` which is only the logo)

daveverwer commented on about/index.md:
```diff
@@ -3,15 +3,14 @@ layout: page
 
-The goal of the Swift project is to create the best available language for uses ranging from systems programming, to mobile and desktop apps, scaling up to cloud services.  Most importantly, Swift is designed to make writing and maintaining *correct* programs easier for the developer. To achieve this goal, we believe that the most obvious way to write Swift code must also be:
+* Swift is general-purpose and modern. Suitable for everything from systems programming, through mobile and desktop apps, to cloud services.
+* Swift is safe.  Undefined behavior is the enemy of safety, and it’s best to catch mistakes should be caught before software is in production. Swift makes the obvious path the safest.
```
Fixed.

daveverwer replied:
Fixed

daveverwer commented:
Current version:

![Screenshot 2023-07-04 at 12 19 24@2x](https://github.com/apple/swift-org-website/assets/5180/a104748b-ddd1-4353-bc7f-211a431c9f58)

About page:

![Screenshot 2023-07-04 at 12 20 15@2x](https://github.com/apple/swift-org-website/assets/5180/3c359112-3a95-45b9-aa2d-50ba270c0829)

alexandersandberg reviewed (approved):
👌

daveverwer commented on about/index.md:
```diff
@@ -3,15 +3,14 @@ layout: page
 
-The goal of the Swift project is to create the best available language for uses ranging from systems programming, to mobile and desktop apps, scaling up to cloud services.  Most importantly, Swift is designed to make writing and maintaining *correct* programs easier for the developer. To achieve this goal, we believe that the most obvious way to write Swift code must also be:
+* Swift is general-purpose and modern. Suitable for everything from systems programming, through mobile and desktop apps, to cloud services.
+* Swift is safe.  Undefined behavior is the enemy of safety, and it’s best to catch mistakes should be caught before software is in production. Swift makes the obvious path the safest.
```
Fixed. Thanks Reda!

daveverwer commented on about/index.md:
```diff
@@ -3,15 +3,14 @@ layout: page
-The goal of the Swift project is to create the best available language for uses ranging from systems programming, to mobile and desktop apps, scaling up to cloud services.  Most importantly, Swift is designed to make writing and maintaining *correct* programs easier for the developer. To achieve this goal, we believe that the most obvious way to write Swift code must also be:
+* Swift is general-purpose and modern. Suitable for everything from systems programming, through mobile and desktop apps, to cloud services.
+* Swift is safe.  Undefined behavior is the enemy of safety, and it’s best to catch mistakes should be caught before software is in production. Swift makes the obvious path the safest.
+* Swift is fast to run and quick to write. Predictable and consistent performance that is as quick as C-based languages, with all the advantages of developer-friendly language design.
```
This is great, thanks. Added.

daveverwer commented:
Final?

![Screenshot 2023-07-05 at 15 21 16@2x](https://github.com/apple/swift-org-website/assets/5180/5b18efc1-5730-40e3-ad56-9ea8c65096a6)

and 


![Screenshot 2023-07-05 at 15 22 59@2x](https://github.com/apple/swift-org-website/assets/5180/d542bbd4-1110-4c9e-8561-6473130f95f0)

tomerd commented:
<3

tomerd commented:
cc @tkremenek @airspeedswift @TimTr
